### PR TITLE
Fix audit_log.configuration.policy argument to `rke_config.services.kube_api` argument

### PR DESCRIPTION
**Pull request by rawmind0**
Fix issue #256

### CODE REVIEW

File: website/docs/r/cluster.html.markdown [hunk 1]
```diff
@@ -43,7 +43,7 @@ resource "rancher2_cluster" "foo-custom" {
             max_size = 100
             path = "-"
             format = "json"
-            policy = jsonencode({"rules":[{"level": "Metadata"}]})
+            policy = jsonencode({"apiVersion":"audit.k8s.io/v1","kind":"Policy","rules":[{"level":"RequestResponse","resources":[{"group":"","resources":["namespaces"]}]}]})
```
Comment (cloudnautique) [inline website/docs/r/cluster.html.markdown]: Can we do this another way, that feels kind of ugly. Should we recommend the user create a file and read in the yaml snippet?

**Review by cloudnautique — CHANGES_REQUESTED** [review]

File: website/docs/r/cluster.html.markdown (hunk 1 above)
Comment (rawmind0) [reply website/docs/r/cluster.html.markdown]: Agreed. Better now??

Comment (cloudnautique) [reply website/docs/r/cluster.html.markdown]: Why are we reading JSON and not YAML? Writing JSON is not fun for a human.

Comment (rawmind0) [reply website/docs/r/cluster.html.markdown]: Agreed, but policy is expected with this format. Tell me if more changes are needed

Comment (rawmind0) [reply website/docs/r/cluster.html.markdown]: Ok, you win. Updated to yaml format. :)

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**